### ISSUE TITLE
Config of lamp strobe option

### ISSUE THREAD

**Issue by drpinball** [BallySternOS/BallySternOS#6]
I noticed that the config file contains two references to the BSOS_SLOW_DOWN_LAMP_STROBE option, this at the beginning:

// Fast boards might need a slower lamp strobe
// #define BSOS_SLOW_DOWN_LAMP_STROBE

And this near the end:

// Fast boards might need a slower lamp strobe
#define BSOS_SLOW_DOWN_LAMP_STROBE  0

This is in the majority of machine configs and my question is do these need to be in twice and does the command default to 'on' or does it need a '1' switch?

In other words is #define BSOS_SLOW_DOWN_LAMP_STROBE  0  'off' i.e. the same as // #define BSOS_SLOW_DOWN_LAMP_STROBE

And #define BSOS_SLOW_DOWN_LAMP_STROBE and #define BSOS_SLOW_DOWN_LAMP_STROBE 1 both 'on'
